manage-teams-identity-mobile-and-desktop/exchange-communication-access-tokens.py

```diff
@@ -0,0 +1,41 @@
+import os
+from azure.communication.identity import CommunicationIdentityClient, CommunicationUserIdentifier
+from msal.application import PublicClientApplication
+
+try:
+   print("Azure Communication Services - Access Tokens Quickstart")
+   # Quickstart code goes here
+   
+   # This code demonstrates how to fetch your Azure AD client ID and tenant ID
+   # from an environment variable.
+   client_id = os.environ["AAD_CLIENT_ID"]
+   tenant_id = os.environ["AAD_TENANT_ID"]
+   authority = "https://login.microsoftonline.com/%s" % tenant_id
+   
+   # Create an instance of PublicClientApplication
+   app = PublicClientApplication(client_id, authority=authority)
+
+   scopes = [ 
+   "https://auth.msft.communication.azure.com/Teams.ManageCalls",
+   "https://auth.msft.communication.azure.com/Teams.ManageChats"
+   ]
+   
+   # Retrieve the AAD token and object ID of a Teams user
+   result = app.acquire_token_interactive(scopes)
```

Comment:
is it possible to configure the redirect uri similarly to other samples? just for the consistency and in case the customer configures a different uri in the portal....

Reply:
https://docs.microsoft.com/en-us/python/api/msal/msal.application.publicclientapplication?view=azure-python here in the prerequisite defined as this and we provided the same uri for .net quickstart

Reply:
:'( the inability to change the path is actually quite unfortunate given the best practice for the redirect uri

```
Do not register multiple redirect URIs where only the port differs. The login server will pick one arbitrarily and use the behavior associated with that redirect URI (for example, whether it's a web-, native-, or spa-type redirect).
```
src: https://docs.microsoft.com/en-us/azure/active-directory/develop/reply-url